src/routes/index.tsx

```diff
@@ -30,7 +30,7 @@ import {
 
 const PrivateRoute = ({ children }: any) => {
   const isOnboarded = useTypedSelector(
-    ({ settings }) => !!(settings.macaroonCredentials && settings.tdexdConnectUrl)
+    ({ settings }) => !!(settings.isInitialized && settings.tdexdConnectUrl)
```

Comment:
`&& settings.tdexdConnectUrl` can be removed

Reply:
`settings.tdexdConnectUrl` is used to determine whether the user is logged in or out.
We should redirect the user to the pairing view when he's not logged in so the check should actually be:
```suggestion
    ({ settings }) => !!(settings.tdexdConnectUrl)
```
The status of the daemon is useless here.